doxygen/contributor_help_pages/require_meta.md

```diff
@@ -0,0 +1,191 @@
+## Using requires<> for general overloads. {#require_meta_doc}
+
+The [`requires`](@ref require_meta) type traits are aliases for `std::enable_if_t` that have premade
+conditions for turning on and off function definitions during compilation.
+These are useful for having generalized templates while still overloading a function or class.
+You can think of these as "legacy concepts" aka I think the closest thing you can get to C++20 concepts without having the `requires` keyword.
+
+`requires` are [`std::enable_if_t`](https://en.cppreference.com/w/cpp/types/enable_if) aliases such as the below definition of @ref stan::require_t.
+
+```cpp
+template <typename T>
+using require_t = std::enable_if_t<T::value>;
+```
+
+Note that unlike `std::enable_if_t`, the alias @ref stan::require_t 's template type `T`
+must have a valid member `value`.
+
+The most common use cases for a `require` is to overload a function or
+declare specializations of a class. For example, the function below will only work
+on types derived from [`Eigen::DenseBase`](https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html) with only 1 row or column at compile time (aka types such as `Eigen::Matrix<double, -1, 1>` or `Eigen::Matrix<double, 1, -1>`).
```

Comment:
aka types such as -> e.g., types such as

(aka is "also known as", "e.g." is "for example", and "i.e." is "in other words".)

Reply:
Fixed